sdk/assembly/__tests__/bignum.spec.ts

```diff
@@ -1,18 +1,22 @@
 import { u128 } from "..";
 
+// TODO: findout why the commented lines below fail
```

Comment:
I assume this TODO comment needs to be addressed prior to merge?

Reply:
Yeah good point.  bignum didn't fail in any of the other tests so I saw this as a future work to understand this edge case.